### ISSUE TITLE
TimeSeriesDataset retrieve columns

### ISSUE THREAD

**Issue by LuisPerezVazquez** [pytorch/pytorch#77736]
### 🚀 The feature, motivation and pitch

Right now it is not clear or easy to retrieve column names from torch tensor after transforming data into pytorch forecasting TimeSeriesDataset. This is crucial to be able to use this library and interpret the results in a more straightforward way.

**Comment by george-qi** [comment]
Hi Luis, thank you for the feature request! Can you provide some more information about this? I'm wondering if you're referring to https://github.com/jdb78/pytorch-forecasting, in which case it would make sense to open an issue with them

**Comment by LuisPerezVazquez** [comment]
Yes, you are right, thanks for the help and sorry for the inconvenience!
